src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,24 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        User bob = new User("email", "password");
+        AuthenticationService authenticationService = (AuthenticationService) injector
+                .getInstance(AuthenticationService.class);
+        try {
+            authenticationService.register(bob.getEmail(), bob.getPassword());
+            User userByLogin = authenticationService.login(bob.getEmail(), bob.getPassword());
+            System.out.println(userByLogin);
+        } catch (RegistrationException | AuthenticationException e) {
+            throw new RuntimeException("Something went wrong", e);
```

Comment:
"Something went wrong" it's not very informative message)

Reply:
fixed